com.jianguoyun.Nutstore.yaml

```diff
@@ -0,0 +1,87 @@
+app-id: com.jianguoyun.Nutstore
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+command: nutstore
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  # So that we can sync files anywhere we want.
+  - --filesystem=home
+separate-locales: false
+
+modules:
+  # Dependency of webkitgtk.
+  - name: unifdef
+    buildsystem: simple
+    build-commands:
+      - make
+      - make prefix=/app install
+    sources:
+      - type: archive
+        url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+        sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+  - name: libsoup2
+    buildsystem: meson
+    config-opts:
+      - -Dtests=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsoup/2.74/libsoup-2.74.3.tar.xz
+        sha256: e4b77c41cfc4c8c5a035fcdc320c7bc6cfb75ef7c5a034153df1413fa1d92f13
+  # Nutstore need libwebkit2gtk-4.0.so, while org.gnome.Platform provides libwebkit2gtk-4.1.so.
+  - name: webkitgtk-4.0
+    buildsystem: cmake
+    config-opts:
+      - -DPORT=GTK
+      - -DENABLE_WEBDRIVER=OFF
+      - -DUSE_LIBBACKTRACE=OFF
+      - -DUSE_GTK4=OFF
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_WAYLAND_TARGET=OFF
+      - -DUSE_SOUP2=ON
+      - -DENABLE_GAMEPAD=OF
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.44.0.tar.xz
+        sha256: c66530e41ba59b1edba4ee89ef20b2188e273bed0497e95084729e3cfbe30c87
+  - name: nutstore
+    buildsystem: simple
+    build-commands:
+      - install -D -m 755 -t /app/bin/ apply_extra nutstore
+      - install -D -t /app/share/icons/hicolor/512x512/apps/ com.jianguoyun.Nutstore.png
```

Comment:
the PNG is 64x64, not 512x512.

Reply:
flatpak lint failed when I install this file to 64x64:
```
{
    "errors": [
        "appstream-missing-icon-file"
    ],
    "message": "Please consult the documentation at https://docs.flathub.org/docs/for-app-authors/linter"
}
```

Reply:
And install it as 512x512 fix it.

Reply:
requirement are 128x128 minimum. CHEATING on the size isn't a solution.

Reply:
The icon is updated to 512x512 now.